packages/nvidia_nat_core/tests/nat/front_ends/auth_flow_handlers/test_websocket_flow_handler.py

```
@@ -168,13 +170,142 @@ async def create_websocket_message(self, msg):
 
     # ----------------- assertions -------------------------------------- #
     assert opened, "The authorization URL was never emitted."
+    assert received_messages[0].use_redirect is False, "Default use_redirect_auth should emit use_redirect=False"
     token_val = ctx.headers["Authorization"].split()[1]
     assert token_val in mock_server.tokens, "token not issued by mock server"
 
     # all flow‑state cleaned up
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)